scripts/objectstore/copy_files_to_irods.py

```diff
@@ -0,0 +1,366 @@
+        print(e)
+        session.cleanup()
+        conn.rollback()
+        read_cursor.close()
+        conn.close()
+        return
+
+    session.cleanup()
+    conn.commit()
+    read_cursor.close()
+    conn.close()
+
+
+def get_irods_resource(conn, objectid, object_store_id, irods_info):
+    try:
+        irods_resc = irods_info["resc"]
+        irods_tape_resc = irods_info["tape_resc"]
+        irods_tape_resc_cuttoff = irods_info["tape_resc_cuttoff"]
+        # Convert string to datetime
+        irods_tape_resc_cuttoff_dt = datetime.strptime(irods_tape_resc_cuttoff, "%m/%d/%Y")
+
+        read_cursor = conn.cursor()
+        args = ('ok', objectid, objectid, tuple([object_store_id]), 'ok', objectid, objectid, tuple([object_store_id]))
+        read_cursor.execute(last_accessed_sql_statement, args)
+        row = read_cursor.fetchone()
+        if row is None:
+            print("Could not find the last access time for dataset with id {}. Returning the default resc {}.".format(objectid, irods_resc))
+            return irods_resc
+
+        dataset_id = row[0]
+        if int(dataset_id) != objectid:
+            print("The returned dataset id {} does not match the passed in datsetid {}. Returning the default resc {}.".format(dataset_id, objectid, irods_resc))
+            return irods_resc
+
+        max_create_time = row[1]
+        # Make max create time offset naive, so it can compared with offset naive tape cuttoff datetime
+        max_create_time_dt = max_create_time.replace(tzinfo=None)
+        # If the last time a dataset was accessed was prior to a cuttoff date, use the tape resource. Otherwise, use the regular (non-tape) resource
+        if max_create_time_dt < irods_tape_resc_cuttoff_dt:
+            print("The last time dataset with id {} was accessed {} is prior to tape resource cuttoff {}. Using tape resource in irods.".format(objectid, max_create_time_dt, irods_tape_resc_cuttoff_dt))
+            return irods_tape_resc
+
+        print("The last time dataset with id {} was accessed {} is after the tape resource cuttoff {}. Using regular (non-tape) resource in irods.".format(objectid, max_create_time_dt, irods_tape_resc_cuttoff_dt))
+        return irods_resc
+
+    except Exception as e:
+        print(e)
+        read_cursor.close()
+        return
+
+
+def get_file_checksum(disk_file_path):
+    checksum_cmd = "shasum -a 256 {} | xxd -r -p | base64".format(disk_file_path)
+    disk_file_checksum = subprocess.check_output(checksum_cmd, shell=True)
+    # remove '\n' from the end of disk_file_checksum
+    disk_file_checksum_len = len(disk_file_checksum)
+    disk_file_checksum_trimmed = disk_file_checksum[0:(disk_file_checksum_len - 1)]
+    # Return Unicode string
+    return disk_file_checksum_trimmed.decode("utf-8")
+
+
+def print_help_msg():
+    print("\nThis script copies datasets from Disk to Irods. It operates on a range of dataset IDs (start_dataset_id and end_dtatset_id, both inclusive).")
```

Comment:
Could you make that a multi-line string please ?
something like
```
help = """
Line 1 and 
Line 2 and so
on
"""
```

Reply:
Sure.